gulpfile.js

```diff
@@ -527,6 +528,79 @@ function buildExperiments(options) {
 
 
 /**
+ * Build "Login Done" page.
```

Comment:
probably wouldn't be `too hard` but wouldn't want you to waste your time on it on this PR. I'll track and work on it. https://github.com/ampproject/amphtml/issues/1346